envoy/README.md

```diff
@@ -0,0 +1,520 @@
+<div align="center">
+<h1>PATH<br/>Authorization & Rate Limiting</h1>
+<img src="https://storage.googleapis.com/grove-brand-assets/Presskit/Logo%20Joined-2.png" alt="Grove logo" width="500"/>
+</div>
+<br/>
+
```

Comment:
@commoddity Wdyt of doing a `TODO_UPNEXT(@commoddity): Document a cheatsheet` to have a `envoy/cheatsheet.md` for a very quick & simple copy-pasta?